openhtf/output/callbacks/mfg_inspector.py

```diff
@@ -148,7 +148,8 @@ def __init__(self,
     if user and keydata:
       self.credentials = oauth2client.client.SignedJwtAssertionCredentials(
           service_account_name=self.user,
-          private_key=self.keydata.encode(),
+          private_key=self.keydata.encode()
```

Comment:
Surround with parentheses and reformat so that the statement starts on the same column across lines.